asterisk16-pbx-lua is missing dependency for liblua5.3.

Hi guys,

`asterisk16-pbx-lua` is failing on master builds.

Faillog from buildbot:
```
Package asterisk16-pbx-lua is missing dependencies for the following libraries:
liblua5.3.so.0.0.0
Makefile:923: recipe for target '/builder/arm_cortex-a9_vfpv3/build/sdk/bin/packages/arm_cortex-a9_vfpv3/telephony/asterisk16-pbx-lua_16.3.0-4_arm_cortex-a9_vfpv3.ipk' failed
make[3]: *** [/builder/arm_cortex-a9_vfpv3/build/sdk/bin/packages/arm_cortex-a9_vfpv3/telephony/asterisk16-pbx-lua_16.3.0-4_arm_cortex-a9_vfpv3.ipk] Error 1
time: package/feeds/telephony/asterisk-16.x/compile#479.90#104.89#863.82
```

The full compile logs can be found here:
https://downloads.openwrt.org/snapshots/faillogs/arm_cortex-a9_vfpv3/telephony/asterisk-16.x/compile.txt
https://downloads.openwrt.org/snapshots/faillogs/aarch64_cortex-a72/telephony/asterisk-16.x/compile.txt
https://downloads.openwrt.org/snapshots/faillogs/powerpc_8540/telephony/asterisk-16.x/compile.txt

--- a/net/asterisk-16.x/Makefile
+++ b/net/asterisk-16.x/Makefile
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 AST_MAJOR_VERSION:=16
 PKG_NAME:=asterisk$(AST_MAJOR_VERSION)
 PKG_VERSION:=$(AST_MAJOR_VERSION).3.0
-PKG_RELEASE:=4
+PKG_RELEASE:=5
 
 PKG_SOURCE:=asterisk-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.asterisk.org/pub/telephony/asterisk/releases
@@ -920,7 +920,7 @@ $(eval $(call BuildAsteriskModule,odbc,ODBC,ODBC support.,+libpthread +libc +uni
 $(eval $(call BuildAsteriskModule,pbx-ael,Asterisk Extension Logic,Asterisk Extension Language compiler.,+$(PKG_NAME)-res-ael-share,extensions.ael,pbx_ael,,))
 $(eval $(call BuildAsteriskModule,pbx-dundi,Dundi,Distributed Universal Number Discovery.,,dundi.conf,pbx_dundi,,))
 $(eval $(call BuildAsteriskModule,pbx-loopback,Loopback switch,Loopback switch.,,,pbx_loopback,,))
-$(eval $(call BuildAsteriskModule,pbx-lua,Lua,Lua PBX switch.,+liblua,extensions.lua,pbx_lua,,))
+$(eval $(call BuildAsteriskModule,pbx-lua,Lua,Lua PBX switch.,+liblua5.3,extensions.lua,pbx_lua,,))
 $(eval $(call BuildAsteriskModule,pbx-realtime,Realtime Switch,Realtime switch.,,,pbx_realtime,,))
 $(eval $(call BuildAsteriskModule,pbx-spool,Call Spool,Outgoing spool support.,,,pbx_spool,,))
 $(eval $(call BuildAsteriskModule,pgsql,PostgreSQL,PostgreSQL support.,+libpq,cel_pgsql.conf cdr_pgsql.conf res_pgsql.conf,cel_pgsql cdr_pgsql res_config_pgsql,,))

